apps/barista/src/layout/smalltile/smalltile.html

```diff
@@ -1,6 +1,6 @@
 <a *ngIf="data" [href]="data.link" class="ba-smalltile-link">
   <div class="ba-smalltile-img">
-    <span>{{ data.identifier }}</span>
+    <span>{{ _identifier() }}</span>
```

Comment:
If the `identifier` is a getter, please use 
```suggestion
    <span>{{ _identifier }}</span>
```